### PR TITLE
Create error fallback page

### PR DESCRIPTION
**Pull request by benny-n**
Provides a permanent solution to the temporary workaround listed in #105

### CODE REVIEW

**Review by liadaram1 — COMMENTED** [review]
It looks fine to me, but maybe worth @NissanOhana will take look as well.

**Comment by benny-n** [comment]
> It looks fine to me, but maybe worth @NissanOhana will take look as well.

@NissanOhana

**Review by NissanOhana — APPROVED** [review]
Hard to follow, but looks good 👍 
I think it would be better if we merge it during the night, and check the deployment if we didn't break anything :) 

image

**Review by NissanOhana — APPROVED** [review]
